Domains/Profiles/Profiles.ecschema.xml

```diff
@@ -366,4 +367,108 @@
         <ECProperty propertyName="Depth" typeName="double" description="Extent of the capsule in the direction of the y-axis." category="ProfileProperties" kindOfQuantity="AECU:LENGTH_SHORT" />
     </ECEntityClass>
+
+    <ECEntityClass typeName="ProfiledSolidDefinitionElement" modifier="Abstract" displayLabel="Profiled Solid Definition Element" description="TODO">
+        <BaseClass>bis:DefinitionElement</BaseClass>
+    </ECEntityClass>
+
+    <ECRelationshipClass typeName="PhysicalTypeUsesProfiledSolidDefinition" modifier="None" strength="referencing" strengthDirection="forward" description="TODO">
+        <BaseClass>bis:ElementRefersToElements</BaseClass>
+        <Source multiplicity="(0..*)" roleLabel="uses" polymorphic="true">
+            <Class class="bis:PhysicalType"/>
+        </Source>
+        <Target multiplicity="(0..1)" roleLabel="is used by" polymorphic="true">
+            <Class class="ProfiledSolidDefinitionElement"/>
+        </Target>
+        <ECProperty propertyName="IsMirrored" typeName="boolean" displayLabel="Is Mirrored" description="TODO" />
+    </ECRelationshipClass>
+
+    <ECRelationshipClass typeName="PhysicalElementUsesProfiledSolidDefinition" modifier="None" strength="referencing" strengthDirection="forward" description="TODO">
+        <BaseClass>bis:ElementRefersToElements</BaseClass>
+        <Source multiplicity="(0..*)" roleLabel="uses" polymorphic="true">
+            <Class class="bis:PhysicalElement"/>
+        </Source>
+        <Target multiplicity="(0..1)" roleLabel="is used by" polymorphic="true">
+            <Class class="ProfiledSolidDefinitionElement"/>
+        </Target>
+        <ECProperty propertyName="IsMirrored" typeName="boolean" displayLabel="Is Mirrored" description="TODO" />
+    </ECRelationshipClass>
+
+    <ECRelationshipClass typeName="ProfiledSolidDefinitionElementUsesProfile" modifier="Abstract" strength="referencing" strengthDirection="forward" description="TODO">
+        <BaseClass>bis:ElementRefersToElements</BaseClass>
+        <Source multiplicity="(0..*)" roleLabel="uses" polymorphic="true">
+            <Class class="ProfiledSolidDefinitionElement"/>
+        </Source>
+        <Target multiplicity="(0..1)" roleLabel="is used by" polymorphic="true">
+            <Class class="Profile"/>
+        </Target>
+        <ECProperty propertyName="CardinalPointName" typeName="string" displayLabel="Cardinal Point" description="TODO" category="ProfiledSolidDefinitionProperties" />
+        <ECProperty propertyName="Offset" typeName="point2d" displayLabel="Offset" description="TODO" category="ProfiledSolidDefinitionProperties" />
+    </ECRelationshipClass>
+
+    <ECEntityClass typeName="ProfiledSolidDefinition" modifier="None" displayLabel="Profiled Solid Definition" description="TODO">
```

Comment:
We have two choices if I understand correctly.  1. Prismatic (constant cross-section) and straight or 2. Varying profile (segmented w/ start and end profiles) w/ general path.  So, if you want constant cross section along an arc, you have you use a segmented profiled solid definition w/ a single segment and same profile at both ends.  Do we want to add a path option w/ single profile?

Reply:
That's a consequence of the simplified "two options" approach. I think you guys concluded that 80% or more of the structural cases fell into #1 (constant cross-section straight). Under such thinking, if you believe the constant cross-section along a non-straight path is common, we'll probably need to revisit such decision and introduce a third case.